crates/router/src/connector/paypal/transformers.rs

```diff
@@ -262,15 +356,44 @@ impl TryFrom<&PaypalRouterData<&types::PaymentsAuthorizeRouterData>> for PaypalP
                 let amount = OrderAmount {
                     currency_code: item.router_data.request.currency,
                     value: item.amount.to_owned(),
+                    breakdown: AmountBreakdown {
```

Comment:
Use From impl for PaymentsAuthorizeRouterData -> AmountBreakdown and remove redundant code